server/views/pages/activities/allocation-dashboard/end-date.njk

```diff
@@ -8,7 +8,11 @@
 {% set pageTitle = applicationName + " - Edit allocation - End date" %}
 {% set pageId = 'edit-allocation-end-date-page' %}
 
-{% set backLinkHref = '/activities/allocation-dashboard/' + allocationId + '/end-date-option' %}
+{% if endDate is undefined %}
```

Comment:
Is the explicit undefined check needed? I think in most places we're simply checking for a falsy value?

E.g., `{% if endDate %}`

Reply:
Done